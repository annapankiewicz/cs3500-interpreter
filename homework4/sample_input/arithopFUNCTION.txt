{
    fuzzy_socks = function(x,y,z) {
	x = 17 + y + z
    };
    clock = 134 * 6 + fuzzy_socks
}
