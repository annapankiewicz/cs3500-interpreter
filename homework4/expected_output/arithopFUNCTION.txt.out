Line 4: Arg 2 cannot be function or null or list
